pkgs/development/tools/code-maat/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  pname = "code-maat";
+  version = "1.0.3";
+
+  src = fetchurl {
+    url = "https://github.com/adamtornhill/code-maat/releases/download/v${version}/code-maat-${version}-standalone.jar";
+    sha256 = "sha256-cAaGX9BX27Z2GN583YmhagWsBIygVc0ZDkzbspM9OJw=";
```

Comment:
```suggestion
    hash = "sha256-cAaGX9BX27Z2GN583YmhagWsBIygVc0ZDkzbspM9OJw=";
```

Reply:
Done

Out of curiosity, what is the rationale for this change?